src/keys.ts

```diff
@@ -15,8 +15,11 @@ export namespace JWTAuthenticationBindings {
   );
 }
 
-export namespace OtherServicesBindings {
-  export const HASH_PASSWORD = BindingKey.create<HashPassword>(
-    'services.hash_password',
+export namespace BcryptHasherBindings {
+  export const BCRYPT_HASHER = BindingKey.create<BcryptHasher>(
+    'services.bcrypt.hasher',
+  );
+  export const ROUND = BindingKey.create<number>(
```

Comment:
```suggestion
  export const ROUNDS = BindingKey.create<number>(
```

(plural "rounds", not a singular "round")